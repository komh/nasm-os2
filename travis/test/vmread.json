[
	{
		"description": "Test vmread instruction",
		"id": "vmread",
		"format": "bin",
		"source": "vmread.asm",
		"target": [
		    { "output": "vmread.bin" },
		    { "stderr": "vmread.ok.stderr" }
		]
	},
	{
		"description": "Test errors in vmread instruction",
		"ref": "vmread",
		"option": "-DERROR",
		"target": [
		    { "stderr": "vmread.err.stderr" }
		],
		"error": "expected"
	}
]
