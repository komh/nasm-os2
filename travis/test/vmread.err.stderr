./travis/test/vmread.asm:8: warning: implicit DEFAULT ABS is deprecated [-w+implicit-abs-deprecated]
./travis/test/vmread.asm:15: error: invalid operand sizes for instruction
./travis/test/vmread.asm:16: error: invalid operand sizes for instruction
./travis/test/vmread.asm:19: error: instruction not supported in 64-bit mode
./travis/test/vmread.asm:20: error: instruction not supported in 64-bit mode
./travis/test/vmread.asm:22: error: invalid operand sizes for instruction
./travis/test/vmread.asm:23: error: invalid operand sizes for instruction
